libs/utils/include/utils/memalign.h

```diff
@@ -17,6 +17,7 @@
 #ifndef TNT_UTILS_MEMALIGN_H
 #define TNT_UTILS_MEMALIGN_H
 
+#include <tuple>
```

Comment:
why is this one needed?

Reply:
For the `std::ignore`, which itself avoid a warn_unused_result

Reply:
okay let's just do `(void)` I don't want to pull a whole header just for that.

Reply:
Ok, done